### PR TITLE
update user, user PATCH, update user table, registration, seeded date

### PR DESCRIPTION
**Pull request by crystaljoyce**
I pulled out incomplete checkout stuff but wanted to update with what I have in the interim.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -90,7 +91,7 @@ const App = () => {
           </Route>
 
           <Route path="/products/:productId">
-            <ProductView />
+            <ProductView selectProduct={setSelectedProduct}/>
```
Comment (samrunyan5) [inline src/components/App.js]: @crystaljoyce this should be` <ProductView selectProduct={selectProduct} setSelectedProduct={setSelectedProduct} />`

File: src/components/App.js [hunk 2]
```diff
@@ -118,7 +119,7 @@ const App = () => {
           </Route>
 
           <Route exact path='/cart/checkout'>
-            <Checkout />
+            <Checkout  />
```
Comment (samrunyan5) [inline src/components/App.js]: I see you have order, user, and component passed in as props in the component. If they are being used, add them in here also.

**Review by samrunyan5 — CHANGES_REQUESTED** [review]
I had a couple of comments for things to be fixed. Otherwise, this looks great. Awesome job with the updateUser. Look at you go!

File: src/components/App.js (hunk 2 above)
Comment (crystaljoyce) [reply src/components/App.js]: Der. Totally forgot about this part. Thank you!

**Pull request closed without merging**